gene_benchmark/descriptor.py

```diff
@@ -648,3 +699,48 @@ def has_missing_columns(df_row, column_names):
 
     """
     return any(missing_col_or_nan(df_row, col) for col in column_names)
+
+
+class BasePairDescriptor(SingleEntityTypeDescriptor):
+    """A descriptor designated to describe each symbol by its base pair sequence."""
+
+    def __init__(self, specie: str = "human", description_col="description") -> None:
+        """
+        Initiates a base pair description object.
+
+        Args:
+        ----
+            specie (str, optional): The species to be used for the base pair sequence. Defaults to "human".
+            description_col (str, optional): Name of the description column. Defaults to "description".
+
+        """
+        self.species = specie
+        self.description_col = description_col
+        self.missing_entities = None
+        self.allow_missing = None
+        self.allow_partial = True
+
+    def _retrieve_dataframe_for_entities(
+        self, entities: list, first_description_only=False
+    ):
+        ensembles = _gene_symbol_to_ensemble_ids(entities, species=self.species)
+        base_pairs = {
```

Comment:
do you think maybe the function should return a dictionary with the ensemble id -> bp ? and insert the loop into the function?

Reply:
I think not; there is the one-purpose principle. We can add a wraper method, but someday we might just want to convert from symbol to ensemble and not have the bp seq.